core/log/file_console_logger.odin

```diff
@@ -6,8 +6,11 @@ import "core:encoding/ansi"
 import "core:fmt"
 import "core:strings"
 import "core:os"
+import "core:os/os2"
```

Comment:
My issue with importing `os2` here is that it doesn't support everything yet. You might have to split `file_console_logger.odin` into two files. One for the platforms that support `os2`, and one for everything else.